README.rst

```diff
@@ -939,6 +961,80 @@ Once you have performed the 3 steps above, you can test it as follows:
+  :alt: Creating a subnet division rule example
+
+2. Create a VPN Server
+~~~~~~~~~~~~~~~~~~~~~~
+
+Now create a VPN Server and choose the previously created **master subnet** as the subnet for
+this VPN Server.
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-controller/issues/400-subnet-subdivision-rule/docs/subnet-division-rule/vpn-server.png
+  :alt: Creating a VPN Server example
+
+3. Create a VPN Client Template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a template, setting the **Type** field to **VPN Client** and **VPN** field to use the
+previously created VPN Server.
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-controller/issues/400-subnet-subdivision-rule/docs/subnet-division-rule/vpn-client.png
+  :alt: Creating a VPN Client template example
+
+**Note**: You can also check the **Enable by default** field if you want to automatically
+apply this template to devices that will register in future.
+
+4. Apply VPN Client Template to Devices
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With everything in place, you can now apply the VPN Client Template to devices.
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-controller/issues/400-subnet-subdivision-rule/docs/subnet-division-rule/apply-template-to-device.png
+  :alt: Adding template to device example
+
+After saving the device, you should see all provisioned Subnets and IPs for this device
+under the `System Defined Variables <#system-defined-variables>`_.
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-controller/issues/400-subnet-subdivision-rule/docs/subnet-division-rule/system-defined-variables.png
+  :alt: Provisioned Subnets and IPs available as System Defined Variables example
+
+Voila! You can now use these variables in configuration of the device. Refer to `How to use configuration variables <#how-to-use-configuration-variables>`_
+section of this documentation to learn how to use configuration variables.
+
+Important Notes
+~~~~~~~~~~~~~~~
+
+- In the above example Subnet, VPN Server, and VPN Client Template belonged to the **default** organization.
+  You can use **Systemwide Shared** Subnet, VPN Server, or VPN Client Template too, but
+  Subnet Division Rule will be always related to an organization. The Subnet Division Rule will only be
+  triggered when such VPN Client Template will be applied to a Device having the same organization as Subnet Division Rule.
+
+- You can also use the configuration variables for provisioned subnets and IPs in the Template.
+  Each variable will be resolved differently for different devices. E.g. ``OW_subnet1_ip1`` will resolve to
+  ``10.0.0.1`` for one device and ``10.0.0.55`` for another. Every device gets its own set of subnets and IPs.
+  But don't forget to provide the default fall back values in the "default values" template field
+  (used mainly for validation).
+
 Default Templates
```

Comment:
I think we should move up default templates and required templates before the sections on variables, because these are more basic features which should be explained first, what do you think?

Reply:
I tried to group it along with other tutorials/how-tos. But you are right, I have reordered them.